democracy/models/section.py

```diff
@@ -67,6 +67,11 @@ def save(self, *args, **kwargs):
                 # This is a new section or changing type from closure info,
                 # automatically derive next ordering, if possible
                 self.ordering = max(self.hearing.sections.values_list("ordering", flat=True) or [0]) + 1
+
+        if self.type == SectionType.objects.get(identifier='introduction'):
```

Comment:
Also, uh, maybe just `if self.type and self.type.identifier == "introduction"`...